tutorontask/templates/ontask/apps/service/settings/base.py

```diff
@@ -480,6 +493,42 @@
 }
 TINYMCE_COMPRESSOR = False
 
+# django-cors-headers
```

Comment:
Why is this section commented?

Reply:
That's how the file comes. No idea: https://github.com/abelardopardo/ontask_b/blob/b69f88d50a85a5a3addfd62b1462ca54249f766f/settings/base.py#L494-L528